.github/workflows/builder_go_slsa3.yml

```diff
@@ -232,7 +245,7 @@ jobs:
 
       - name: Upload generated binary
         id: upload
-        uses: slsa-framework/slsa-github-generator/.github/actions/secure-upload-artifact@e3220805577deb9d193f64e519abcb3b50851df5
+        uses: ./__BUILDER_CHECKOUT_DIR__/.github/actions/secure-upload-artifact
```

Comment:
my earlier comment: after the buidl procedure, do we trust `./__BUILDER_CHECKOUT_DIR__`?
https://github.com/slsa-framework/slsa-github-generator/issues/880#issuecomment-1282771342

Reply:
Got it, resolved in the issue!